### PR TITLE
feat: Bump cbwp from alpha to beta

### PR DESCRIPTION
**Pull request by yuwenma**

### CODE REVIEW

File: pkg/test/resourcefixture/testdata/basic/cloudbuild/v1beta1/cloudbuildworkerpool/_generated_object_cloudbuildworkerpool.golden.yaml
Comment (yuwenma) [inline pkg/test/resourcefixture/testdata/basic/cloudbuild/v1beta1/cloudbuildworkerpool/_generated_object_cloudbuildworkerpool.golden.yaml:1]: Updated the update.yaml to verify all mutable field can be updated.

**Review by justinsb — APPROVED** [review]

**Comment by justinsb** [comment]
/hold

In case we don't want to go the beta yet

**Pull request merged**